src/schema/datatypes/photo.yaml

```diff
@@ -0,0 +1,13 @@
+group1:
+  datatypes:
+    - meg
+    - eeg
+    - ieeg
+  suffixes:
+    - photo
```

Comment:
Dear @tsalo -- **Thank you** for leading this effort!  It is really fun to look at BIDS in such as structured way!

So within this `datatypes/` we indeed have datatypes which gained their `folder/` within BIDS hierarchy (they also have themselves duplicated now in `datatypes`) such as `anat/`, `func/`, `fmap/`, etc. (+ `beh/` which isn't imaging), and then we have another group -- those which could have indeed been a modality of their own, but they are not "imaging data". They became auxiliary files which could go along with base imaging (and `beh/`) `datatypes`.  I think that 
- all the `photo`, `channels` etc should get their own "folder" in this schema, e.g. `auxiliary` or  `auxdatatypes`, and be moved there (instead of being under `datatypes`)
- `datatypes/` entries can get rid of the `datatypes` field in their .yaml files -- they should then have 1-to-1 correspondence from the filename to corresponding datatype.

On the top level (for inclusion) I see smth like
```yaml
datatypes:
  anat: !include datatypes/anat.yaml
  func: !include datatypes/func.yaml
  ...
auxdatatypes:
  channels: !include auxdatatypes/channels.yaml
  event: !include auxdatatypes/events.yaml
```
This would make it easier to tell one (which gets its own folder in BIDS) from another (just provides additional suffixed files).

Alternative could be - keep as is, and then add additional field  (in a yet to be created higher level "inclusion" file) to signal which datatype is `auxiliary`, but then it would be a bit less structured IMHO and most likely we would need to maintain that redundant self mentioning within `datatypes` for the base data types.

Reply:
I split `auxdatatypes` from `datatypes` and dropped the `datatypes` key from the `datatype` yamls. I have to say, it looks much cleaner.